src/pages/sym.mdx

```diff
@@ -0,0 +1,132 @@
+An implementation may also allow the bank to be the case-insensitive keyword `BOOT` to specify symbols in the loaded boot ROM.
+If the bank is omitted, then the colon must be as well; the results of such lines are implementation-defined.
+
+The address must be in 16-bit range (0 to $FFFF inclusive), the bank must be in 32-bit range (0 to $FFFFFFFF inclusive); implementations may abandon processing the line if either number is larger than these limits.
+Implementations may accept bank numbers higher than $FFFFFFFF.
+Nonsensical locations (e.g. `2:9800`) may produce a warning.
+
+Two locations are considered identical if they have the same bank and the same address.
+Two distinct symbols can have identical locations; in particular, <LatestStableLink to="rgbasm.5#Unions">unions</LatestStableLink> facilitate this.
+
+Locations are strictly ordered by memory region (ROM0, ROMX, VRAM, SRAM, WRAM0, WRAMX, Echo RAM, OAM, I/O, HRAM, IE), then by bank, then by address.
+Entries may or may not be sorted in the sym file; for example, RGBLINK before v0.4.0 output symbols in the same order that it read them from its input files.
+Some sym files are piped through [`sort(1)`](https://man.openbsd.org/sort.1), which often produces files sorted by bank then address, but not by region.
+
+Sym files do not carry information about the memory region a symbol belongs to, but that can be inferred from the address most of the time.
+This detection is sometimes ambiguous: for example, `0:A000` could be one past the end of VRAM, or at the beginning of SRAM.
+The following heuristics are suggested:
+- If the symbol's bank is in range of exactly one of the candidate regions, assign it to that one. (*There should be no ambiguity, e.g. `2:A000`.*)
+- If the symbol is local and is "attached" to a global symbol (see the "presentation shortening" below), assign it to the same region as the global symbol. (*Local labels should "follow" the global symbol they are attached to, if any.*)
+- If the symbol's name ends with (case-insensitive) "end", assign it to the earlier region. (*This is a common naming convention, and "one-past-end" labels are typically "end markers".*)
+- If all else fails, assign it to the later region (e.g. `0:A000` would be assigned to RAM). (*This is correct more often than not in practice.*)
+
+##### Name
+
+The symbol name can contain any characters except ASCII spaces, tabs, non-printable ASCII characters, or characters excluded by earlier processing (e.g. semicolons).
+A symbol whose name does not contain any periods (`.`, U+002E) is a **global symbol**.
+If a symbol's name contains exactly one period, it is a **local symbol**; the portion up to but not including the period is the **global portion**, the rest is the **local portion**; neither shall be empty.
+A symbol with two or more periods in its name shall be processed in an implementation-defined manner, which may include rejecting or ignoring it.
```

Comment:
Define this. No reason not to.

Reply:
Sure, what should it be defined to, then?

Reply:
Whatever you want RGBASM to do with such a symbol in the future, even if it's not implemented today.

Reply:
...which involves a lot of moving parts that I don't want to design ahead of time.